packages/ketcher-react/src/script/ui/state/hotkeys.ts

```diff
@@ -76,7 +76,7 @@ function keyHandle(dispatch, state, hotKeys, event) {
       const newAction = actions[actName].action
       const hoveredItemId = getHoveredAtomId(render.ctab.atoms)
       const isHoveringOverAtom = hoveredItemId !== null
-      if (isHoveringOverAtom) {
+      if (isHoveringOverAtom && newAction.tool !== 'select') {
```

Comment:
Could you, please, also remove logic for select tool from `handleHotkeyOverAtom`?